packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx

```diff
@@ -34,10 +34,10 @@ export const SwitchMemberModal = () => {
 
   const filteredMembers = useMemo(() => {
     if (modalData?.membersToShow) {
-      return members.filter((member) => modalData.membersToShow?.includes(member.id))
+      return members.filter((member) => modalData.membersToShow?.includes(member.id)).sort((a,b) => (a.handle > b.handle) ? 1 : ((b.handle > a.handle) ? -1 : 0))
```

Comment:
The main problem with this way is that sorts uppercase char first, so `Z` comes before `a`. There is a `Comparator` function in `packages/ui/src/common/model/Comparator.ts` which should work well here:
```suggestion
      return members.filter((member) => modalData.membersToShow?.includes(member.id)).sort(Comparator(false, 'handle'))
```
(it just uses `string.localCompare(string)` under the hood)